vyper/ast/signatures/function_signature.py

```diff
@@ -120,12 +120,7 @@ def set_default_args(self):
     # Get a signature from a function definition
     @classmethod
     def from_definition(
-        cls,
-        func_ast,  # vy_ast.FunctionDef
-        global_ctx,
-        interface_def=False,
-        constant_override=False,  # CMC 20210907 what does this do?
```

Comment:
let's try to do keep all refactoring of FunctionSignature in one place going forward, see https://github.com/vyperlang/vyper/pull/3390#issuecomment-1538452240